### PR TITLE
GCI-35 - Sign in page

### PR DESCRIPTION
**Pull request by Izaron**
Worked on "Sign in page"
JIRA issue: https://issues.openmrs.org/browse/GCI-35
Melange task:
https://www.google-melange.com/gci/task/view/google/gci2014/4619183002222592

### CODE REVIEW

File: app/system-modules/auth/views/login.ejs [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
-		</span>
-	</div>
-	
-	<div class="field">
-		<input class="btn btn-success" type="submit" value="Login with OpenMRS ID &#187;">
-	</div>
-</form>
\ No newline at end of file
+<div class="panel panel-default">
+  <div class="panel-body" style="padding-bottom:50px">
```
Comment (elliottwilliams) [inline app/system-modules/auth/views/login.ejs]: Could you remove the extra padding here? I think it will help the panel content look more "balanced"

**Comment by elliottwilliams** [comment]
Looks good, thanks for your contribution! Once we resolve that line in the diff I'll accept.

**Comment by elliottwilliams** [comment]
Thanks!

**Comment by robbyoconnor** [comment]
Why is the branch gone?

**Pull request closed without merging**